addon/components/o-s-s/chip.hbs

```diff
@@ -0,0 +1,4 @@
+<div class={{this.computedClass}} ...attributes>
```

Comment:
aren't we supposed to show a tooltip if there is a max width?

Reply:
`We must also add a prop to define a fixed max-width to the component with an ellipsis on the text and a tooltip (show on hover) to show the full label.`

Reply:
tooltip is now in component but ...attributes wasn't remove because if we want , we can add attribute